apps/frontend/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,60 @@
+import { type ReactNode, useEffect, useRef } from "react";
+
+import crossIconSrc from "~/assets/images/icons/cross.svg";
+
+import styles from "./modal.module.css";
+
+type Properties = {
+	children: ReactNode;
+	onClose: () => void;
+	open: boolean;
+	title: string;
+};
+
+const Modal = ({ children, onClose, open, title }: Properties): JSX.Element => {
+	const dialogReference_ = useRef<HTMLDialogElement>(null);
```

Comment:
Why there is underscore?
```suggestion
	const dialogReference = useRef<HTMLDialogElement>(null);
```